contracts/src/v0.8/ccip/test/BaseTest.t.sol

```diff
@@ -35,7 +35,7 @@ contract BaseTest is Test {
 
   // Onramp
   uint96 internal constant MAX_NOP_FEES_JUELS = 1e27;
-  uint96 internal constant MAX_MSG_FEES_JUELS = 1e18;
+  uint96 internal constant MAX_MSG_FEES_JUELS = 1_000e18;
```

Comment:
more realistic value, some router test was failing when using the 1.6 onRamp and 1e18 as max fee